jobs/aws/eks-distro/release-1-18-postsubmits.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+postsubmits:
+  aws/eks-distro:
+  - name: release-1-18-postsubmit
+    always_run: false
+    run_if_changed: "release/1-18/RELEASE"
+    max_concurrency: 10
+    cluster: "prow-postsubmits-cluster"
+    branches:
+    - ^main$
+    skip_report: false
+    decoration_config:
+      gcs_configuration:
+        bucket: s3://prowdataclusterstack-316434458-prowbucket7c73355c-1n9f9v93wpjcm
+        path_strategy: explicit
+      s3_credentials_secret: s3-credentials
+      timeout: "4.0h"
+    labels:
+      image-build: "true"
+    spec:
+      serviceaccountName: postsubmits-release-account
+      automountServiceAccountToken: false
+      containers:
+      - name: build-container
+        image: 316434458148.dkr.ecr.us-west-2.amazonaws.com/eks-distro/builder:bb13cd7056bb2a6582ff7ea96d99e597bffefaee
+        command:
+        - bash
+        - -c
+        - >
+          make setup
```

Comment:
Can you link what this is doing? Otherwise I might prefer having a different target than `release` like `public-release` or something

Reply:
https://github.com/aws/eks-distro/blob/d2459e56dd6b551a2093238790a295d809aa1bdd/Makefile#L24

Reply:
Yea I still prefer sticking to one target rather than reference multiple targets in the Prow job

Reply:
We should have one make call because we want to pass all the same arguments to it

Reply:
Alternatively, setup can be the first dependency on release